CRUD-APP/component/Home.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import {Link} from 'react-router-dom';
+
+const Home = () =>{
+
+    const [users,setUser] =useState([]); 
+
+    useEffect(() =>
+    {
+        loadUsers();
+    } ,[]);
+
+    const loadUsers = async() =>{
+        const result= await axios.get("http://localhost:3003/users");
+        setUser(result.data.reverse());
+    };
+   
+    const deleteUser = async id =>{
+        await axios.delete(`http://localhost:3003/users/${id}`);

```

Comment:
when making API calls try to use trycatch block to catch an error and show error to user.